front/app/modules/commercial/machine_translations/hooks/useTranslation.ts

```diff
@@ -28,23 +30,32 @@ export default function useTranslation({
       locale_to: localeTo,
       attribute_name: attributeName,
     },
-    enabled: isMachineTranslationsEnabled && context === 'initiative',
+    enabled:
+      machineTranslationButtonClicked &&
+      isMachineTranslationsEnabled &&
```

Comment:
Just to be sure, the translate buttons will still be hidden when the translation feature is disabled?